### DISCUSSION TITLE
Welcome to fractus-atlas Discussions!

### DISCUSSION THREAD

**Discussion by Jimskapt** [Jimskapt/fractus-atlas#19] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ghost** [comment]
Sorry if i just am stupid. But HOW do you install the mac version? As far as i can see it just a file, which isn't an installer...

**Reply by Jimskapt** [reply]
Hi @OWL4C,

I apology, I think the problem should come from my build and tests process : I've always automatically built this program with GitHub Actions for MAC and I never have opportunity to really test it on a functionning MAC.

So, despite it share same code and functionalities as Windows and Linux (I've tested them), I can't be impossible that the MAC build never worked.

By the way, fractus-atlas is designed to be as it is called an "portable application" so don't need to be installed. You have just to "run this executable" (the file) and it should take care of all and open the window to use it. Without the need to stay indefinitely on your system.

I hope you don't have loosed too much time on this problem. If you are still interested and ready to do it, maybe we can find a way to understand the problem when you run it on your MAC ?

**Reply by ghost** [reply]
The problem is that the program doesn't execute but the OS tells me that there is no program installed to handle this filetype.

**Reply by Jimskapt** [reply]
Hello @OWL4C,

Looks like a common way to install an Application on MacOS is to mount a *.dmg file and then drag & drop its content into `Applications` special system folder.

Have you tried to directly drag & drop the downloaded file `fractus-atlas-1.14.0-macos-x86_64` into this `Applications` folder and then run it from this folder ?

If it does not not works, are you using the new ARM M1 processor, or are you using a good old Intel x86 processor (as 64 bits) ?
